app/components/publishers/vacancies_component.rb

```diff
@@ -18,8 +18,13 @@ def render?
     @organisation.all_vacancies.active.any?
   end
 
-  def selected_class(vacancy_type)
-    "govuk-tabs__list-item--selected" if @selected_type == vacancy_type
+  def is_selected(vacancy_type)
+    @selected_type == vacancy_type
+  end
+
+  def vacancy_links
+    @vacancy_links = @vacancy_types
+    @vacancy_links.map { |vacancy_type| vacancy_type_tab_link(vacancy_type, is_selected(vacancy_type)) }
```

Comment:
I reckon this could just be:

```ruby
@vacancy_types.map { |vacancy_type| vacancy_type_tab_link(vacancy_type, is_selected(vacancy_type)) }
```

`#map` will return a copy of the original array (not modify it in place), so I don't think the separate instance variable `@vacancy_links` is needed.